backend/src/api.rs

```diff
@@ -88,18 +80,12 @@ async fn songs(req: HttpRequest, query: web::Query<HashMap<String, String>>) ->
                 s.title,
                 s.filetype,
                 s.song_media,
-                g.id,
-                g.name,
                 a.id,
                 a.name
             FROM
                 songs as s
-            INNER JOIN song_genre as sg
-                ON s.id = sg.song_id
             INNER JOIN song_artist as sa
                 ON s.id = sa.song_id
-            INNER JOIN genres as g
-                ON g.id = sg.genre_id
```

Comment:
The same as above - everything involved with genres should be added again in the next sprint